connect/src/main/java/com/stripe/android/connect/analytics/ConnectAnalyticsEvent.kt

```diff
@@ -43,8 +43,8 @@ internal sealed class ConnectAnalyticsEvent(
      */
     data class WebComponentLoaded(
         val pageViewId: String,
-        val timeToLoad: Double,
-        val perceivedTimeToLoad: Double
+        val timeToLoad: Long,
+        val perceivedTimeToLoad: Long
```

Comment:
Same here. Even better, could you add `@param` docs for all the events? It's not obvious what many of the params mean, nor their shape.

Reply:
➕ I have comments documenting these in the [Analytic Events](https://docs.google.com/document/d/1nzHGofoclzij4nP5n5qgSpYLia-93heStXvt7vOfrkQ/edit?tab=t.0#bookmark=id.hbdcbks7wn2n) section of our scoping doc. I think you could copy these over verbatim (that's what I did on iOS)